packages/openapi-to-graphql/src/types/options.ts

```diff
@@ -110,14 +116,21 @@ export type Options = {
    */
   requestOptions?: NodeRequest.OptionsWithUrl
 
+  /**
+   * Allows to override or add options to the PubSub connect object used to make
+   * publish/subscribe to the API backend.
+   * e.g. Setup the web proxy to use.
+   */
+  connectOptions?: ConnectOptions
```

Comment:
Create an option object that would follow publish / subcribe semantics instead of response / request